packages/dashboard/src/components/map-app.tsx

```diff
@@ -310,7 +309,15 @@ export const MapApp = styled(
       const sub = rmf.fleetsObs
         .pipe(
           switchMap((fleets) =>
-            merge(...fleets.map((f) => (f.name ? rmf.getFleetStateObs(f.name) : EMPTY))),
+            merge(
+              ...fleets.map((f) =>
+                f.name
+                  ? rmf
+                      .getFleetStateObs(f.name)
+                      .pipe(throttleTime(5000, undefined, { leading: true, trailing: true }))
```

Comment:
Can we have the throttle time to be 1000?
5000 makes the robots rather glitchy and jump around every 5 seconds. The motion of the robot is quite important for remote monitoring so I think 1000 milliseconds is good, while every other subscription we can keep at 5000.

Reply:
reduced to 500 and remove unused code